mmv1/products/dataplex/Datascan.yaml

```diff
@@ -0,0 +1,640 @@
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      Description of the scan.
+  - !ruby/object:Api::Type::String
+    name: 'displayName'
+    description: |
+      User friendly display name.
+  - !ruby/object:Api::Type::KeyValuePairs
+    name: 'labels'
+    description: |
+      User-defined labels for the scan. A list of key->value pairs.
+  - !ruby/object:Api::Type::Enum
+    name: 'state'
+    output: true
+    description: |
+      Current state of the DataScan.
+    values:
+      - :STATE_UNSPECIFIED
+      - :ACTIVE
+      - :CREATING
+      - :DELETING
+      - :ACTION_REQUIRED
+  - !ruby/object:Api::Type::String
+    name: 'createTime'
+    output: true
+    description: |
+      The time when the scan was created.
+  - !ruby/object:Api::Type::String
+    name: 'updateTime'
+    output: true
+    description: |
+      The time when the scan was last updated.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'data'
+    required: true
+    immutable: true
+    description: |
+      The data source for DataScan.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'entity'
+        immutable: true
+        description: |
+          The Dataplex entity that represents the data source(e.g. BigQuery table) for Datascan.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'executionSpec'
+    description: |
+      DataScan execution settings.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: 'trigger'
+        description: |
+          Spec related to how often and when a scan should be triggered. If not specified, the default is OnDemand, which means the scan will not run until the user calls dataScans.run API.
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: 'onDemand'
+            description: |
+              The scan runs once via dataScans.run API.
+          - !ruby/object:Api::Type::NestedObject
+            name: 'schedule'
+            description: |
+             The scan is scheduled to run periodically.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: 'cron'
+                required: true
+                description: Cron schedule for running scans periodically. This field is required for Schedule scans.
+      - !ruby/object:Api::Type::String
+        name: 'field'
+        immutable: true
+        description: |
+          The unnested field (of type Date or Timestamp) that contains values which monotonically increase over time. If not specified, a data scan will run for all data in the table.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'executionStatus'
+    output: true
+    description: |
+      Status of the data scan execution.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'latestJobEndTime'
```

Comment:
should we make this and its sibling `output: true` as well?

Reply:
+1